### DISCUSSION TITLE
Weblate instead of Crowdin

### DISCUSSION THREAD

**Discussion by comradekingu** [Floorp-Projects/Floorp#912] [Feature Request]
Mozilla uses Pontoon, which has its own issues as it is based on Crowdin, but
there are many projects doing well with Weblate.
There are zero projects doing well with Crowdin, because it doesn't respect locales,
it has no multi-language view, and it isn't possible to span large arrays of strings at once, and so on.
Currently using LibreWolf to get away from privacy issues, so it is all in tune with what Floorp seems to be about :)

**Comment by creeper-0910** [comment]
Crowdin has an OSS license, but weblate does not appear to have one.
If Firefox is using a Crowdin-based version, i think it should use a similar one.

**Reply by comradekingu** [reply]
Weblate has this license https://github.com/WeblateOrg/weblate/blob/main/COPYING
Crowdin is closed source proprietary only-as-a-service software.

If you mean does any of them have hosting under preferable terms,
Weblate has https://hosted.weblate.org/hosting/
Crowdin has hosting for non-commercial open source projects, and says it is gratis for some academic endeavours.

Mozilla made something that looks like Crowdin, and similarly it is a bad tool.

**Reply by creeper-0910** [reply]
Sorry, by OSS license I mean the Crowdin license for OSS software developers.

It is expensive to host Weblate on an ongoing basis.
Therefore, using Crowdin is more reasonable for OSS projects.

**Reply by comradekingu** [reply]
Crowdin only has a license for non-commercial OSS software developer.
Weblate has https://hosted.weblate.org/hosting/ available at no monetary cost, and it is
what most projects not tied to their own instance uses.

Crowdin is neither good as a tool, or as a practice. Read the terms and conditions. Grim stuff.

**Reply by creeper-0910** [reply]
Hmmm...
We have no plans to migrate at this time, but if hosted 
weblate is free, I will consider it. (As far as I can tell, it is listed as costing at least € 17 per month.)

**Reply by regularentropy** [reply]
I believe it would be more advantageous for translators to keep the translation process on Crowdin. The platform boasts a significantly larger community, increasing our chances of attracting qualified translators without requiring them to register or search for a separate instance.

t. maintainer of english, russian and ukrainian tranlsations

**Reply by comradekingu** [reply]
Translators being subjected to the terms and conditions of Crowdin is least of all good for translators.
I would argue Crowdin has zero community, and much less any traction with libre software.
There is a userbase, but it comprises of accounts doing not much tied to existing projects.
So you get a fairly random assortment of people clicking buttons in no unified manner.
The resulting drive-by translations means lots of users with no overview, which Crowdin as a platform facilitates.
Other projects see little benefit from this, as there is no discovery feature of note.

The best translators have never been on Crowdin, but some are also there.
To my knowledge not a single project has had success translating anything on Crowdin.
The difference between assumed chance and active management favours a platform that allows it.

**Reply by maboroshin** [reply]
comradekingu writes complete fantasy. Here is a summary of objective data. https://github.com/maboroshin/translation/issues/1

**Reply by comradekingu** [reply]
I'll just link to my own reply of your other attempt, which in addition makes it clear
you don't actually know the functionality of the platforms discussed.
https://github.com/NightscoutFoundation/xDrip/discussions/3420#discussioncomment-10506586

**Comment by CutterKnife** [comment]
If the specs are the same as Crowdin's, it would be very cumbersome to have to re-translate from scratch in all languages...

**Reply by comradekingu** [reply]
The already translated files can be imported on any new platform, so translations are not lost.
The idea is to gain in both function and avoid the loss of privacy.
Putting eggs into a basket controlled entirely by a third-party is not beneficial.

**Reply by maboroshin** [reply]
An approval system is in operation for some languages on Crowdin. Maybe, it's time-consuming to reproduce this.

comradekingu uses abstract language and does not talk about Weblate privacy issues. Here is some objective data to help you rethink. https://github.com/maboroshin/translation/issues/1

**Comment by Telmesomething** [comment]
@comradekingu,

Slightly off topic, but Librewolf is not so privacy-conscious as you may think. See [this analysis](https://www.unixsheikh.com/articles/choose-your-browser-carefully.html#librewolf) for more info.
